### ISSUE TITLE
Other POCs

### ISSUE THREAD

**Issue by JimCircadian** [alan-turing-institute/IceNet-Project#19]
Use this issue to track other potential POC integrations/adoptions that can be followed up as time/project development allows

**Comment by JimCircadian** [comment]
I suggest marking these complete only once activity is undertaken and tracked by a new ticket. 

* [ ] [Discuss Pangeo usage with MS and add dataset as part of the planetary computer (consider public licensing issues alongside this)](https://planetarycomputer.microsoft.com/applications)

**Comment by JimCircadian** [comment]
This is not in our scope to deal with until further developments are planned. 

Closing as part of the POC project wrap up
